workspace.json

```diff
@@ -749,11 +749,6 @@
       "sourceRoot": "typings",
       "tags": ["platform:any"]
     },
-    "@fluentui/public-docsite-v9": {
```

Comment:
this should not be removed.  workpsace.json is the only source of truth of all existing projects in monorepo. has no reliance on any underlying technology like lerna, yarn workspaces etc. please add this back.